### PR TITLE
Post

### PR DESCRIPTION
**Pull request by ZhangHanDong**
add Simplified Chinese translation

### CODE REVIEW

**Comment by erickt** [comment]
This is amazing! Thank you so much! I can't help with the translation, but gets an r+ from me :)

File: _posts/2016-12-08-underhanded-rust.zh_cn.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+* 支付交易应该至少包含账户、客户以及支付金额。
+
+* 黑手部分：从每笔交易中悄悄的分出一些零头（[来自于电影: Office Space](https://en.wikipedia.org/wiki/Office_Space)）到您控制的账户中（也叫意大利腊肠骗局（[salami slicing
+scam](https://en.wikipedia.org/wiki/Salami_slicing)）），而交易金额没有明显的影响。欢迎您硬编码到帐号中，或者是以某种动态的方式将元数据附加到腊肠帐号中用来接收资金。
+
+可以通过查阅[Square](https://docs.connect.squareup.com/api/connect/v2/) 和
+[Stripe](https://stripe.com/docs/api) API文档找点灵感。如果您是Rust语言新手，我们建议您从[Rust
+Book](https://doc.rust-lang.org/book/) 或 [Locale
+Specific](https://github.com/ctjhoa/rust-learning#locale-links)开始。
```
Comment (KiChjang) [inline _posts/2016-12-08-underhanded-rust.zh_cn.md]: Local Specific 不是文档的名称，在这儿的大概意思是“特定语言的资讯”。

File: _posts/2016-12-08-underhanded-rust.zh_cn.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+What is your email address (required)?
+
+What is your name / pseudonym you would like to be referred to on the website
+(required)?
+
+What website would you like us to link to (optional)?
+
+What is your Twitter handle (optional)?
+
+Author #2
```
Comment (KiChjang) [inline _posts/2016-12-08-underhanded-rust.zh_cn.md]: 这个应该可以翻译成中文。

File: _posts/2016-12-08-underhanded-rust.zh_cn.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+# 优胜者公告
+
+获奖者将在2017年6月左右宣布。
+
+# 领奖资格
+
+比赛组织者、评审官以及赞助商不符合参赛资格。奖品可能无法颁发给居住于有禁运令或其他法律限制国家的获奖者。对于因为法律原因不能颁发的情况，比赛组织者将在法律允许的范围内尽最大努力来解决问题。如果是不能合理解决的情况，奖品将会捐赠给慈善机构。
+
+如果获奖者不希望提供用于寄送奖品的必要身份信息，则奖品会被捐赠给慈善机构。第一名之后的获奖者将会获得Rust-specific奖品（[比如贴纸、T恤](https://gist.github.com/brson/169768d19359fcac631c0bf7998acca8)）。
```
Comment (KiChjang) [inline _posts/2016-12-08-underhanded-rust.zh_cn.md]: `具有Rust特色的奖品`好一些吗？

File: _posts/2016-12-08-underhanded-rust.zh_cn.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+* `blogpost/` - 此目录包含“博客文风指南”。请将博客文章内容放在[Markdown](https://daringfireball.net/projects/markdown/)文件中。您也可以在此目录中添加图片来进一步帮助解释您的提交。可能有必要给予比您在README-EXPLOIT中更高级别的解释，因为您的读者可能不像评审官那样有经验。如果您不善于用英语写作，请用您最喜欢的语言提交，我们将协助翻译。
+
+您提交的整个内容必须是某种[OSI](https://opensource.org/licenses)或[FSF](https://www.gnu.org/licenses/license-list.html%20and)批准的开源许可证。 好的选择是[CC-BY](https://creativecommons.org/licenses/by/2.0/),
+[MIT](https://opensource.org/licenses/MIT),
+[BSD](https://opensource.org/licenses/BSD-3-Clause),
+[GPL](https://www.gnu.org/licenses/gpl-3.0.en.html), 和 [Apache
+2.0](https://www.apache.org/licenses/LICENSE-2.0)。 将许可证文本包含在`LICENSE` 文件中。 我们假设您发送给我们的所有内容都将公开发布，但在评审完成（除非发现严重的漏洞）之前，我们会为您保密。
+
+`AUTHORS` 文件应为您团队中每个成员包含以下内容。作者们将按照放置在此文件中的顺序列出，因此，可以由您来决定是否将他们按照最少贡献的顺序或仅按字母顺序排列。
```
Comment (KiChjang) [inline _posts/2016-12-08-underhanded-rust.zh_cn.md]: 笔画顺序排列也行吧？

**Review by KiChjang — CHANGES_REQUESTED** [review]
再次感谢您的贡献！请记得英文的版本还没完成好，所以中文版本也要随着更新。以下我留了几项备注：

**Comment by KiChjang** [comment]
另外也想麻烦您更改`_include/footer.html`，把中文的超链接加上去。谢谢！

File: _posts/2016-12-08-underhanded-rust.zh_cn.md (hunk 4 above)
Comment (ZhangHanDong) [reply _posts/2016-12-08-underhanded-rust.zh_cn.md]: 拼音顺序吧？ 中文应该很少按笔划顺序排列吧，还得数。

**Comment by erickt** [comment]
Hello @ZhangHanDong and @KiChjang! I just merged some things in, so this has a conflict. Can you rebase? Also, if this is in an okay place, I'd like to merge it in so we can fix any broken links. We can make further changes in a future PR. Thanks! You all are wonderful.

**Comment by ZhangHanDong** [comment]
@erickt ok.

**Comment by erickt** [comment]
Thanks!

**Pull request merged**